tests/data_context/test_data_context_datasources.py

```diff
@@ -16,7 +17,7 @@
 from great_expectations.data_context.types.base import (
     DataContextConfig,
     DatasourceConfig,
-    GXCloudConfig,
+    GXCloudConfig, InMemoryStoreBackendDefaults,
```

Comment:
I think the linter may get mad here - you'll want to run `ruff --fix .` or set up pre-commit hooks to cover this